examples/config.hocon.sample

```diff
@@ -86,6 +86,8 @@ streams {
 s3 {
   region = "{{s3Region}}"
   bucket = "{{s3bucket}}"
+  # optional bucket where to store partitioned data
+  bucketJson = "{{s3bucket}}/partitioned"
```

Comment:
Would be nice to precise that this needs to be specified if we want the data to be partitioned by type. Or use a boolean for that like `partitionByType`.